acceptance/tests/facts/verify_tmpfs_file_system.rb

```diff
@@ -0,0 +1,52 @@
+# This test is intended to demonstrate that mount resource can mount tmpfs file systems
+# and the mount facter mountpoints should show the mount as tmpfs
+test_name 'C98163: mountpoints fact should show mounts on tmpfs' do
+  tag 'risk:high'
+
+  require 'facter/acceptance/user_fact_utils'
+  extend Facter::Acceptance::UserFactUtils
+
+  confine :except, :platform => 'windows'
```

Comment:
So many `confine :except`. Time to make this `confine, :platform => blah` instead?